tools/acorn-optimizer.js

```diff
@@ -190,7 +199,7 @@ function restoreForVars(node) {
   let restored = 0;
   function fix(init) {
     if (init && init.type === 'EmptyStatement') {
-      assert(init.oldDeclarations);
+      assertAt(init.oldDeclarations, init.start);
```

Comment:
Why isn't this
```suggestion
      assertAt(init.oldDeclarations, init);
```
? That is, I'd expect to see a node used and the location to mention as that same node, not a subnode of it.

Reply:
`start` and `end` are node attributes that refer to offsets within the input text.   i.e this is the offset of the node within the file, not a subnode.

Reply:
Oh, I see. Do we not always want the start? It seems simpler to pass in the node, and let `assertAt` take `.start` - we might also want to show the whole range actually, up to the end?

Reply:
Done